lib/utils/parameter_utils.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:analyzer/dart/ast/ast.dart';
+
+/// Checks if parameter has underscores in its name
+bool hasNoUnderscoresInName(FormalParameter parameter) =>
+    parameter.name != null &&
+    parameter.name!.lexeme.replaceAll('_', '').isNotEmpty;
```

Comment:
Let's avoid `!`.
```suggestion
/// Checks if parameter has underscores in its name
bool hasNoUnderscoresInName(FormalParameter parameter) {
    final paramName = parameter.name
    if (paramName == null) return false; // That's what we would get from your version, but it seems odd. Should it be `true`?
    return paramName.lexeme.replaceAll('_', '').isNotEmpty;
    }
```

Reply:
Also, positive checks are usually easier to reason about. We can always negate it later, while negating an already negated function to create a positive check is weird to read.
hasNoUnderscoresInName -> hasUnderscoresInName

Reply:
Wait, why isn't this a `String.contains()` check for that matter? 
It really checks whether the param consists _entirely_ of underscores.

Reply:
Function originates from DCM, and it's name gave me confusion it actually checks if param name consists only from underscores. Will rename and implement changes

Reply:
`String.contains('_')` will return true for '_param' and 'par_am', so we need to either count number of underscores and check it with string length or replace all underscores and look is something left or not. I've choose the latter.